app/config/dev-config-pub.yaml

```diff
@@ -0,0 +1,36 @@
+projectId: dartlang-pub-dev
```

Comment:
Let's add this in two phase: in this PR we keep the hard-coded values in the `configuration.dart` as is, and also add this file + an additional test to check that the content of the file is indeed the same as the value in `configuration.dart`.

In a follow-up PR we can remove/cleanup the parts in `configuration.dart`.